.github/workflows/rust.yml

```
@@ -104,7 +92,7 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: test
-          args: --all --no-fail-fast
+          args: --all --no-fail-fast --doc
```

As above, probably use the `dtolnay/rust-toolchain@stable` and then:
```suggestion
          run:
            cargo test --no-fail-fast
```